Tables/Tab81752.NSPEntryTable3.al

```diff
@@ -0,0 +1,24 @@
+table 81753 "NSP Entry Table3"
```

Comment:
@NAVFreak 
Probably a new table should be added to PermissionSet

Reply:
You are correct